runner/src/scenario/ceramic/model_instance.rs

```diff
@@ -251,6 +274,20 @@ impl CeramicModelInstanceTestUser {
                 )
                 .await?
             }
+            super::ReuseType::LeadWorkerSubscriber => {
+                // For model instance reuse type make it work the same way as shared
+                Self::generate_mids(
+                    user,
+                    &config.user_cli,
+                    &config.redis_cli,
+                    &small_model_id,
+                    &large_model_id,
+                    config.params.number_of_documents,
+                    global_leader,
```

Comment:
This should be `true` so that each worker creates MIDs instead of just the global leader.
```suggestion
                    true,
```

Reply:
Basically, we need the `ReuseType::Shared` code when creating models, but `ReuseType::PerUser` code when creating MIDs.